uaclient/entitlements/base.py

```diff
@@ -153,6 +166,78 @@ def can_enable(self, silent: bool = False) -> bool:
             if not silent:
                 print(details)
             return False
+
+        if self.incompatible_services:
+            return self.handle_incompatible_services()
+
+        return True
+
+    def handle_incompatible_services(self):
+        """
+        Handle to proceed when a incompatible service is found during enable
+
+        When enabling a service, we may find that there is an incompatible
+        service already enable. In that situation, we can ask the user
+        if the incompatible service should be disabled before proceeding.
+        There are also different ways to configure that behavior:
+
+        We can disable removing incompatible service during enable by
+        adding the following lines into uaclient.conf:
+
+        features:
+          block_disable_on_enable: true
+
+        We can also use the --allow-disable flag during enable to
```

Comment:
I think this docstring doesn't reflect the code right now, but I feel like we **should** provide a separate cmdline flag for this condition because auto-disabling livepatch without prompt represents dropping security patch updates to the system and that is a significant decisision that we don't really want to apply to the system in the FIPS case because there is no easy way back from that on a system given that we've modified the openssl stack.


This use-case feels quite a bit like apt command-line's `--assume-yes` versus `--force-yes` options since livepatch -> FIPS transition is potentially a harmful/irreparable config change on the system.

```bash
  --force-yes
       Force yes; this is a dangerous option that will cause apt to
       continue without prompting if it is doing something potentially
       harmful. It should not be used except in very special situations.
   -y, --yes, --assume-yes
       Automatic yes to prompts; assume "yes" as answer to all prompts and
       run non-interactively. If an undesirable situation, such as
       changing a held package, trying to install an unauthenticated
       package or removing an essential package occurs then apt-get will
       abort. Configuration Item: APT::Get::Assume-Yes.
```

Reply:
We tabled this in discussion as FIPS-users providing the --assume-yes are ones that really know what they are doing anyway and wouldn't want the expectation of livepatch on the system introducing deltas anyway.